app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -72,7 +69,6 @@ def update
     api :POST, '/organizations', N_('Create organization')
     param :name, String, :desc => N_("name"), :required => true
     param :label, String, :desc => N_("unique label")
-    param :description, String, :desc => N_("description")
```

Comment:
@iNecas if I remove the description from here will it be shown properly in apidocs with katello as @thomasmckay asked?